src/js/apps/globals/nav_app.js

```diff
@@ -154,28 +161,45 @@ export default App.extend({
   },
   stateEvents: {
     'change:currentApp': 'onChangeCurrentApp',
+    'change:isMinimized': 'onChangeIsMinimized',
   },
   viewEvents: {
     'click:addPatient': 'onClickAddPatient',
+    'click:minimizeMenu': 'onClickMinimizeMenu',
   },
   childApps: {
     search: SearchApp,
   },
   selectNav(appName, event, eventArgs) {
     this.setState('currentApp', appName);
 
-    const navMatch = this.getNavMatch(appName, event, compact(eventArgs));
+    this.navMatch = this.getNavMatch(appName, event, compact(eventArgs));
```

Comment:
Switch this to `this.navMatch` so it can be used in the `onChangeIsMinimized()` function.